integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -310,7 +310,10 @@ public void startRangerContainer() {
       synchronized (ContainerSuite.class) {
         if (rangerContainer == null) {
           // Start Ranger container
-          RangerContainer.Builder rangerBuilder = RangerContainer.builder().withNetwork(network);
```

Comment:
I will help you push your new Ranger Docker image into the docker hub.
So, please roll back these changes.